source/18_docker_and_containers.rst

```diff
@@ -0,0 +1,196 @@
+Docker and Containers
+=====================
+
+*New and Improved!*
+-------------------
+* Configuration management is approaching 15 years old.
+* Some of the command line utilities we use date back to times when people used
```

Comment:
I mean to emphasize that everything else we've discussed in the course is old, sometimes *really old*.